descriptions/api.github.com/api.github.com.json

```diff
@@ -51967,6 +52070,106 @@
           "enabledForGitHubApps": false,
           "previews": [
 
+          ],
+          "category": "packages",
+          "subcategory": null
+        }
+      },
+      "delete": {
+        "summary": "Delete package version for a user",
+        "description": "Deletes a specific package version for a user. If the package is public and the package version has more than 5,000 downloads, you cannot delete the package version. In this scenario, contact GitHub support for further assistance.\n\nTo use this endpoint, you must authenticate using an access token with the `packages:read` and `packages:delete` scopes. In addition:\n- If `package_type` is not `container`, your token must also include the `repo` scope.\n- If `package_type` is `container`, you must also have admin permissions to the container you want to delete.",
+        "tags": [
+          "packages"
+        ],
+        "operationId": "packages/delete-package-version-for-user",
```

Comment:
same as https://github.com/github/rest-api-description/pull/504/files#r688059211 please set `operationId` to `packages/delete-package-version-for-authenticated-user`